MoviesExample/app/src/main/java/com/novoda/demo/movies/MainActivity.java

```diff
@@ -38,16 +38,25 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onMovieSelected(final Movie movie) {
                 startLoadingTrailer(movie);
             }
-
-            @Override
-            public void onPageLoadRequested(final int page) {
-                moviesViewModel.loadMore();
-            }
         });
         resultList.setAdapter(adapter);
 
         viewDataBinding.setViewmodel(moviesViewModel);
         viewDataBinding.setLifecycleOwner(this);
+
+        moviesViewModel.getPaginatedMovies().observe(this, new Observer<PagedList<Movie>>() {
```

Comment:
Ideally, we could use a [BindingAdapter](https://developer.android.com/reference/android/databinding/BindingAdapter) but I am not sure if this is possible since we need the `LifecycleOwner`.